masonite/auth/Sign.py

```diff
@@ -38,11 +38,18 @@ def sign(self, value):
         
         Returns:
             string -- Returns the encrypted value.
+
+        Raises:
+            InvalidSecretKey -- Thrown if the secret key has incorrect padding.
         """
 
-        f = Fernet(self.key)
-        self.encryption = f.encrypt(bytes(value, 'utf-8'))
-        return self.encryption.decode('utf-8')
+        try:
+            f = Fernet(self.key)
+            self.encryption = f.encrypt(bytes(value, 'utf-8'))
+            return self.encryption.decode('utf-8')
+        except binascii.Error:
+            raise InvalidSecretKey("The encryption key passed does not have correct padding and therefore cannot be used as a secret key.")
+
```

Comment:
It's technically OK to just wrap the `f = Fernet(self.key)` key in the try catch and not the decode method since it the `f = Fernet(self.key)` line of code always executes first and therefore will always throw the exception before it hits the decode. If it ever hits the decode method then it can't throw an exception because the secret key is good. Does that make sense?

```python
        try:
            f = Fernet(self.key)
        except binascii.Error:
            raise InvalidSecretKey("The encryption key passed does not have correct padding and therefore cannot be used as a secret key.")

        self.encryption = f.encrypt(bytes(value, 'utf-8'))
        return self.encryption.decode('utf-8')
```

The awesome part about tests is you can restructure your code in anyway you want and as long as the tests pass then you're still good. So doing that small change above still makes the tests pass